### PR TITLE
Todo-List with add, remove and interaction

### PR DESCRIPTION
**Pull request by Smart1-hub**
Add a new JavaScript file and import it as a module:
Add event listener to the checkbox (change).
Update items object's value for completed key upon user actions.
Implement a function for the "Clear all completed" button.
Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

image

### CODE REVIEW

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Smart1-hub , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Even though you have done a good job by including the optional requirement. The todo items don't seem to be showing on the screen even thought they are getting added to the localstorage. Please fix this.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Smart1-hub — COMMENTED** [review]
All source (src) files: html, css and js have been bundled, through webpack, to distribution (dist) files where the output is stored.

File: src/index.js [hunk 1]
```diff
@@ -30,26 +31,25 @@ const editTask = (desc, index) => {
...
-  if (checkbox.checked === true) {
-    tasks[index].checked = true;
-  } else {
-    tasks[index].checked = false;
-  }
-};
-
 const displayTasks = () => {
   taskWrapper.innerHTML = '';
+  taskWrapper.classList.add('drag-list');
```
Comment (mikethreels) [inline src/index.js:36]: - [ ] the to-do tasks are still not displayed on the screen of the user, this issue originates from the fact that you are looking for `.to-dos` class in the `index.js` however when running `npm run build` and checking the `index.html` in the dist folder there is not tag with this class name. therefore it is not able to find anything.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Ademola, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Smart1-hub — COMMENTED** [review]
Class name in the src/index.html file has been renamed appropriately

**Review by t-yanick — APPROVED** [review]
Hi @Smart1-hub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**